### PR TITLE
script/build-integration-branch: automatically update local master

### PR DESCRIPTION
**Pull request by xiexingguo**
Signed-off-by: xie xingguo <xie.xingguo@zte.com.cn>

- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

### CODE REVIEW

**Comment by xiexingguo** [comment]
@liewegas I am still a fan of the `build-integration-branch` tool :-). Can we get this minor improvement in?

File: src/script/build-integration-branch [hunk 1]
```diff
@@ -42,6 +42,14 @@ for issue in j:
     prtext.append(pr['html_url'] + ' - ' + pr['title'])
 print("--- queried %s prs" % len(prs))
 
+# update local master
+print('--- checkout master')
+r = call('git checkout master', shell=True)
+assert not r
+print('--- pull master from upstream')
+r = call('git pull upstream master', shell=True)
```
Comment (tchaikov) [inline src/script/build-integration-branch]: in my case, the ceph/ceph repo is not named `upstream`. not sure about other developers.

Comment (xiexingguo) [reply src/script/build-integration-branch]: I made the assumption based on [1], which is widely used among the core developers.
For your case, I think we could simply add a alias, e.g.:
```
$ git remote -v
ceph    https://github.com/ceph/ceph.git (fetch)
ceph    https://github.com/ceph/ceph.git (push)
origin  https://xiexingguo@github.com/xiexingguo/ceph.git (fetch)
origin  https://xiexingguo@github.com/xiexingguo/ceph.git (push)
upstream        https://github.com/ceph/ceph.git (fetch)
upstream        https://github.com/ceph/ceph.git (push)
```

I wouldn't mind if there is any other solutions as long as they can achieve the same effect...

----
[1] https://github.com/ceph/ceph/blob/master/src/script/ptl-tool.py#L15

**Comment by liewegas** [comment]
Have you tried ptl-tool.py?  I think the --base master argument does exactly what you're looking for...

**Pull request closed without merging**